src/lib/util.ts

```diff
@@ -109,7 +109,7 @@ export function blobToArrayBuffer(blob: Blob): Promise<ArrayBuffer> {
   return new Promise((resolve) => {
     const fileReader = new FileReader();
     fileReader.addEventListener('load', () => {
-      resolve(fileReader.result);
+      resolve(fileReader.result! as ArrayBuffer);
```

Comment:
What's this bit doing?

Reply:
Since TypeScript v3, result can be a string, null or an ArrayBuffer.

Reply:
I’ll add some handling